indico/modules/events/static/templates/emails/paper_judgment_notification_email.txt

```diff
@@ -0,0 +1,39 @@
+{% extends 'emails/base.txt' %}
+
+{% macro get_comments() %}
+    {% filter dedent -%}
+        Please see the comments below from the reviewing team:
+
+        "{{ paper.judgment_comment }}"
+    {%- endfilter %}
+{% endmacro %}
+
+{% block subject %}Your paper "{{contribution.title}}" for "{{contribution.event_new.title}}" has been completely reviewed{% endblock %}
```

Comment:
Personally, I don't like the wording so much. How about
_"The reviewing process of your paper ... for ... has finished"_?